app/routes/circulars/circulars.server.ts

```diff
@@ -110,6 +110,62 @@ export async function search({
 
   const [startTime, endTime] = getValidDates(startDate, endDate)
 
+  const esQuery = query
+    ? {
+        function_score: {
+          query: {
+            bool: {
+              must: [
+                {
+                  query_string: {
+                    query: `*${query}*`,
```

Comment:
The wildcards before and after will do unexpected and unpredictable things if `query` actually consists of multiple terms or Lucene syntax.

Reply:
accounted for this